### DISCUSSION TITLE
Please update AngleSharp dependency

### DISCUSSION THREAD

**Discussion by hromkes** [mganss/HtmlSanitizer#538] [General]
Please update AngleSharp dependency

**Comment by mganss** [comment]
Done. 8.1.860-beta.

**Comment by JeffDunbarRPM** [comment]
@mganss is there a timeline on when this will be moved to the main version?

**Reply by mganss** [reply]
No, it's dependent upon the release of AngleSharp.Css 1.0. See https://github.com/AngleSharp/AngleSharp.Css/milestone/2
